src/sc/embedder/tests/test_content.py

```diff
@@ -151,8 +148,8 @@ def test_vimeo_oembed(self):
             add_form.widgets['IDublinCore.title'].value, u'The Backwater Gospel')
         self.assertNotEqual(
             add_form.widgets['IDublinCore.description'].value, u'')
-        self.assertEqual(add_form.widgets['width'].value, u'1280')
-        self.assertEqual(add_form.widgets['height'].value, u'720')
+        self.assertEqual(add_form.widgets['width'].value, u'480')
```

Comment:
from time to time vimeo api behaves different.. we already change it other times.. I don't know why..